File: TransCoder_tokenized_test_set_functions/java/FIND_THE_MINIMUM_DISTANCE_BETWEEN_TWO_NUMBERS.java
int minDist ( int arr [ ], int n, int x, int y ) {
  int i, j;
  int min_dist = Integer . MAX_VALUE;
  for ( i = 0;
  i < n;
  i ++ ) {
    for ( j = i + 1;
    j < n;
    j ++ ) {
      if ( ( x == arr [ i ] && y == arr [ j ] || y == arr [ i ] && x == arr [ j ] ) && min_dist > Math . abs ( i - j ) ) min_dist = Math . abs ( i - j );
    }
  }
  return min_dist;
}

File: results/py/FIND_THE_MINIMUM_DISTANCE_BETWEEN_TWO_NUMBERS.cpp
int f_filled ( int * arr , int n , int x , int y ) {
  int f_filled = 99999999 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    for ( int j = i + 1 ;
    j < n ;
    j ++ ) {
      if ( ( x == arr [ i ] && y == arr [ j ] || y == arr [ i ] && x == arr [ j ] ) && f_filled > abs ( i - j ) ) f_filled = abs ( i - j ) ;
    }
    return f_filled ;
  }
}
